sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/SessionCatalog.scala

```diff
@@ -880,6 +880,26 @@ class SessionCatalog(
     }
   }
 
+  /**
+   * List all matching views in the specified database, including global/local temporary views.
+   */
+  def listViews(db: String, pattern: String): Seq[TableIdentifier] = {
+    val dbName = formatDatabaseName(db)
+    val dbViews = if (dbName != globalTempViewManager.database) {
+      requireDbExists(dbName)
+      externalCatalog.listViews(dbName, pattern).map { name =>
+        TableIdentifier(name, Some(dbName))
+      }
+    } else {
+      Nil
+    }
+    // Always get global/local temp views
```

Comment:
Since this is also based on pattern, we don't get result 'Always'.
`Always get` -> `Always try to get` or we can remove this comment.